analyzer/src/main/kotlin/Analyzer.kt

```diff
@@ -144,11 +125,56 @@ class Analyzer(private val config: AnalyzerConfiguration) {
 
         val endTime = Instant.now()
 
-        val run = AnalyzerRun(startTime, endTime, Environment(), config, analyzerResultBuilder.build())
+        val run = AnalyzerRun(startTime, endTime, Environment(), config, analyzerResult)
 
         return OrtResult(repository, run)
     }
 
+    @ObsoleteCoroutinesApi
+    private suspend fun analyzeInParallel(
+        managedFiles: Map<PackageManager, List<File>>,
+        packageCurationsFile: File?
+    ): AnalyzerResult {
+        val dispatcher = newFixedThreadPoolContext(5, "storage")
```

Comment:
The name should be "analyse" or similar.

Reply:
Done.